routers/repo/blame.go

```diff
@@ -94,6 +94,16 @@ func RefBlame(ctx *context.Context) {
 		log.Error("GetLatestCommitStatus: %v", err)
 	}
 
+	// Check LFS Lock
+	isLFSLock := false
+	lfsLock, err := ctx.Repo.Repository.GetTreePathLock(ctx.Repo.TreePath)
```

Comment:
These changes should be moved inside `if ctx.Repo.CanEnableEditor() {` below (line 132). No need to read and check locks if editor can not be enabled anyway

Reply:
Done.